1-sum-app/sum-app.css

```diff
@@ -0,0 +1,44 @@
+.wrapper-content{
+
```

Comment:
Please avoid these extra spaces. I think we need to find some better css formatter plugin.